packages/shared/src/hooks/useCreateSquadModal.ts

```diff
@@ -34,9 +34,10 @@ export const useCreateSquadModal = ({
     });
   const openSquadBetaModal = () =>
     openModal({
-      type: LazyModal.BetaSquad,
+      type: LazyModal.NewSquad,
```

Comment:
We should also remove the betaSquad from the useModal props.

Reply:
If it is not going to be used anywhere, then I agree, we should remove it.

Reply:
You can just delete it since it was only used here anyways. And if that changes in the future for some reason, we can re-add it

Reply:
Awesome, I will do that then. Thanks for the confirmation.